src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -325,9 +325,10 @@ doclet.help.search.head=Search
 # Introduction to Javadoc search features, followed by a list of examples
 doclet.help.search.intro=You can search for definitions of modules, packages, types, fields, methods, \
     system properties and other terms defined in the API, using some or all of the name, optionally \
-    using "camelCase" abbreviations. For example:
+    using "camelCase" abbreviations. As of JDK 19, JavaDoc also supports searching for multiple terms. \
+    For example:
```

Comment:
Not a fan of version-specific phrasing. Also, "multiple terms" is very (very) vague.

One possibility is to merge with the previous sentence:
```
using some or all of the name, optionally \
     using "camelCase" abbreviations, or a series of parts of the term, separated by whitespace. \
```

Even better, to prevent an excessively long sentence, stop the first sentence after "API", which then describes what you can search for, and then start a new sentence after that, to describe how you can search for those items.